cdap-ui/app/features/etlapps/etlapps.less

```diff
@@ -158,6 +158,50 @@ body.theme-cdap.state-adapters-list {
       padding-left: 5px;
       padding-right: 5px;
     }
+    .label.etlappcontainer {
+      position: absolute;
+      top: 0;
+      right: 0px;
+      // top: 13px;
+      // right: 7px;
+      font-size: 13px;
+      background-color: @cdap-darkblue;
+      // transform: rotate(36deg);
+
+      &.green {
+        background-color: @brand-success;
+        // top: 20px;
+        top: 0;
+        right: 0;
+        // right: 2px;
+        &:after {
+          border-bottom: 5px solid @brand-success;
+        }
+      }
+
+      // &:before {
```

Comment:
This will *probably* used by @astewes or not to style the tags for an etl adapter in the list view. I have done very basic stylings to the tag.